pkg/workload/admissionchecks.go

```diff
@@ -128,8 +127,7 @@ func SetAdmissionCheckState(checks *[]kueue.AdmissionCheckState, newCheck kueue.
 // RejectedChecks returns the list of Rejected admission checks
 func RejectedChecks(wl *kueue.Workload) []kueue.AdmissionCheckState {
 	rejectedChecks := make([]kueue.AdmissionCheckState, 0, len(wl.Status.AdmissionChecks))
-	for i := range wl.Status.AdmissionChecks {
-		ac := wl.Status.AdmissionChecks[i]
+	for _, ac := range wl.Status.AdmissionChecks {
```

Comment:
nit: I prefer to revert to avoid distractions

Reply:
Done